deparse.c

```diff
@@ -1845,6 +1853,60 @@ sqlite_deparse_range_tbl_ref(StringInfo buf, PlannerInfo *root, RelOptInfo *fore
 										 params_list);
 }
 
+/*
+ * Add a RETURNING clause, if needed, to an INSERT/UPDATE/DELETE.
+ */
+static void
```

Comment:
Could you add definition for this function (prototype)?

Reply:
Please verify, @lamdn1409 !

Reply:
Please rename the function with the prefix sqlite_.

Reply:
Done, please confirm, @lamdn1409. I see this prefix.